enterprise/enrollment_api.py

```diff
@@ -37,3 +37,11 @@ def enroll_user_in_course(user, course_details, mode):
     """
     client = get_enrollment_api_client()
     return client.enrollment.post(dict(user=user, course_details=course_details, mode=mode))
+
+
+def get_enrolled_courses(user):
+    """
+    Query the enrollment API to get a list of the courses a user is enrolled in.
+    """
+    client = get_enrollment_api_client()
```

Comment:
@haikuginger it might be a good time to refactor this api into a class - repeating `get_enrollment_api_client` in every method is a certain code smell.